db/migrate/20200304113719_remove_price_in_pence_from_attachments.rb

```diff
@@ -0,0 +1,5 @@
+class RemovePriceInPenceFromAttachments < ActiveRecord::Migration[5.1]
+  def change
+    remove_column :attachments, :price_in_pence, :integer
```

Comment:
Can we combine these two things into the same migration?

Reply:
👍